cmd/check.go

```diff
@@ -12,32 +12,32 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// checkCmd represents the check command
+type CheckCommandFlags struct {
+	CatalogSource          string `json:"catalogsource"`
+	CatalogSourceNamespace string `json:"catalogsourcenamespace"`
+}
+
+var checkflags CheckCommandFlags
+
+// TODO: provide godoc compatible comment for checkCmd
 var checkCmd = &cobra.Command{
-	Use:   "check",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Use: "check",
+	// TODO: provide Short description for check command
+	Short: "",
+	// TODO: provide Long description for check command
+	Long: ``,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("check called")
-		capability.OperatorInstallAllFromCatalog("certified-operators", "openshift-marketplace")
+		capability.OperatorInstallAllFromCatalog(checkflags.CatalogSource, checkflags.CatalogSourceNamespace)
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(checkCmd)
+	flags := checkCmd.Flags()
 
-	// Here you will define your flags and configuration settings.
-
-	// Cobra supports Persistent Flags which will work for this command
-	// and all subcommands, e.g.:
-	// checkCmd.PersistentFlags().String("foo", "", "A help for foo")
-
-	// Cobra supports local flags which will only run when this command
-	// is called directly, e.g.:
-	// checkCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	flags.StringVar(&checkflags.CatalogSource, "catalogsource", "",
+		"")
+	flags.StringVar(&checkflags.CatalogSourceNamespace, "catalogsourcenamespace", "",
```

Comment:
Just a small suggestion - Can we provide `openshift-marketplace` as the default CatalogSourceNamespace if the flag is not provided?